Source/Core/LongLatGeocoderService.js

```diff
@@ -0,0 +1,58 @@
+/*global define*/
+define([
+    './Cartesian3',
+    './defaultValue',
+    '../ThirdParty/when'
+], function(
+    Cartesian3,
+    defaultValue,
+    when) {
+    'use strict';
+
+    /**
+     * Provides geocoding through Bing Maps.
+     *
+     * @alias LongLatGeocoderService
+     * @constructor
+     */
+    function LongLatGeocoderService() {
```

Comment:
I went with `Cartographic` here since that's what's currently used in Cesium and it's also slightly less cumbersome than `LongitudeLatitude`.